docs/integrations/amazon-aws/sqs.md

```diff
@@ -157,20 +156,14 @@ json "userIdentity", "eventSource", "eventName", "awsRegion", "recipientAccountI
 ```
 
 ## Centralized AWS CloudTrail Log Collection
-
 In case you have a centralized collection of CloudTrail logs and are ingesting them from all accounts into a single Sumo Logic CloudTrail log source, create the following **Field Extraction Rule** to map a proper AWS account(s) friendly name/alias. Create it if not already present/update it as required.
 
-**Rule Name**: AWS Accounts
-
-**Applied at**: Ingest Time
-
-**Scope (Specific Data)**: _sourceCategory=aws/observability/cloudtrail/logs
-
-**Parse Expression**:
+* **Rule Name**: AWS Accounts
+* **Applied at**: Ingest Time
+* **Scope (Specific Data)**: _sourceCategory=aws/observability/cloudtrail/logs
 
 Enter a parse expression to create an “account” field that maps to the alias you set for each sub account. For example, if you used the “dev” alias for an AWS account with ID "528560886094" and the “prod” alias for an AWS account with ID "567680881046", your parse expression would look like:
-
-```
+```sql title="Parse Expression"
```

Comment:
Parse expression should be outside code block. Just as it is on the main branch.